---
description: Размещение веб-содержимого в приложении Win32 с помощью элемента управления Microsoft Edge WebView2
title: Microsoft Edge WebView2 для приложений Win32
author: MSEdgeTeam
ms.author: msedgedevrel
ms.date: 06/05/2020
ms.topic: reference
ms.prod: microsoft-edge
ms.technology: webview
keywords: IWebView2, IWebView2WebView, webview2, WebView, приложения Win32, Win32, EDGE, ICoreWebView2, ICoreWebView2Controller, элемент управления "веб-браузер", HTML Edge
ms.openlocfilehash: b7d738628d6627232780e003f126b45c0421d6f0
ms.sourcegitcommit: 8dca1c1367853e45a0a975bc89b1818adb117bd4
ms.translationtype: MT
ms.contentlocale: ru-RU
ms.lasthandoff: 06/08/2020
ms.locfileid: "10699127"
---
# <span data-ttu-id="d12a6-104">Класс Microsoft. Web. WebView2. Core. EdgeNotFoundException</span><span class="sxs-lookup"><span data-stu-id="d12a6-104">Microsoft.Web.WebView2.Core.EdgeNotFoundException class</span></span> 

<span data-ttu-id="d12a6-105">Пространство имен: Microsoft. Web. WebView2. Core </span><span class="sxs-lookup"><span data-stu-id="d12a6-105">Namespace: Microsoft.Web.WebView2.Core</span></span>\
<span data-ttu-id="d12a6-106">Сборка: Microsoft. Web. WebView2. Core. dll</span><span class="sxs-lookup"><span data-stu-id="d12a6-106">Assembly: Microsoft.Web.WebView2.Core.dll</span></span>

```
class Microsoft.Web.WebView2.Core.EdgeNotFoundException
  : public Exception
```

<span data-ttu-id="d12a6-107">Исключение, возникающее при отсутствии установки Edge.</span><span class="sxs-lookup"><span data-stu-id="d12a6-107">The exception that is thrown when an Edge installation is missing.</span></span>

## <span data-ttu-id="d12a6-108">Краткий обзор</span><span class="sxs-lookup"><span data-stu-id="d12a6-108">Summary</span></span>

 <span data-ttu-id="d12a6-109">Участников</span><span class="sxs-lookup"><span data-stu-id="d12a6-109">Members</span></span>                        | <span data-ttu-id="d12a6-110">Описания</span><span class="sxs-lookup"><span data-stu-id="d12a6-110">Descriptions</span></span>
--------------------------------|---------------------------------------------
[<span data-ttu-id="d12a6-111">EdgeNotFoundException</span><span class="sxs-lookup"><span data-stu-id="d12a6-111">EdgeNotFoundException</span></span>](#edgenotfoundexception) | <span data-ttu-id="d12a6-112">Инициализирует новый экземпляр класса EdgeNotFoundException.</span><span class="sxs-lookup"><span data-stu-id="d12a6-112">Initializes a new instance of the EdgeNotFoundException class.</span></span>
[<span data-ttu-id="d12a6-113">EdgeNotFoundException</span><span class="sxs-lookup"><span data-stu-id="d12a6-113">EdgeNotFoundException</span></span>](#edgenotfoundexception) | <span data-ttu-id="d12a6-114">Инициализирует новый экземпляр класса EdgeNotFoundException ссылкой на внутреннее исключение, которое является причиной этого исключения.</span><span class="sxs-lookup"><span data-stu-id="d12a6-114">Initializes a new instance of the EdgeNotFoundException class with a reference to the inner exception that is the cause of this exception.</span></span>
[<span data-ttu-id="d12a6-115">EdgeNotFoundException</span><span class="sxs-lookup"><span data-stu-id="d12a6-115">EdgeNotFoundException</span></span>](#edgenotfoundexception) | <span data-ttu-id="d12a6-116">Инициализирует новый экземпляр класса EdgeNotFoundException с указанным сообщением об ошибке.</span><span class="sxs-lookup"><span data-stu-id="d12a6-116">Initializes a new instance of the EdgeNotFoundException class with a specified error message.</span></span>
[<span data-ttu-id="d12a6-117">EdgeNotFoundException</span><span class="sxs-lookup"><span data-stu-id="d12a6-117">EdgeNotFoundException</span></span>](#edgenotfoundexception) | <span data-ttu-id="d12a6-118">Инициализирует новый экземпляр класса EdgeNotFoundException с указанным сообщением об ошибке и ссылкой на внутреннее исключение, которое является причиной этого исключения.</span><span class="sxs-lookup"><span data-stu-id="d12a6-118">Initializes a new instance of the EdgeNotFoundException class with a specified error message and a reference to the inner exception that is the cause of this exception.</span></span>

## <span data-ttu-id="d12a6-119">Участников</span><span class="sxs-lookup"><span data-stu-id="d12a6-119">Members</span></span>

#### <span data-ttu-id="d12a6-120">EdgeNotFoundException</span><span class="sxs-lookup"><span data-stu-id="d12a6-120">EdgeNotFoundException</span></span> 

<span data-ttu-id="d12a6-121">Инициализирует новый экземпляр класса EdgeNotFoundException.</span><span class="sxs-lookup"><span data-stu-id="d12a6-121">Initializes a new instance of the EdgeNotFoundException class.</span></span>

> <span data-ttu-id="d12a6-122">общедоступная [EdgeNotFoundException](#edgenotfoundexception)()</span><span class="sxs-lookup"><span data-stu-id="d12a6-122">public [EdgeNotFoundException](#edgenotfoundexception)()</span></span>

#### <span data-ttu-id="d12a6-123">EdgeNotFoundException</span><span class="sxs-lookup"><span data-stu-id="d12a6-123">EdgeNotFoundException</span></span> 

<span data-ttu-id="d12a6-124">Инициализирует новый экземпляр класса EdgeNotFoundException ссылкой на внутреннее исключение, которое является причиной этого исключения.</span><span class="sxs-lookup"><span data-stu-id="d12a6-124">Initializes a new instance of the EdgeNotFoundException class with a reference to the inner exception that is the cause of this exception.</span></span>

> <span data-ttu-id="d12a6-125">общедоступный [EdgeNotFoundException](#edgenotfoundexception)(внутренние исключения)</span><span class="sxs-lookup"><span data-stu-id="d12a6-125">public [EdgeNotFoundException](#edgenotfoundexception)(Exception inner)</span></span>

##### <span data-ttu-id="d12a6-126">Параметры</span><span class="sxs-lookup"><span data-stu-id="d12a6-126">Parameters</span></span>
* `inner` <span data-ttu-id="d12a6-127">Исключение, вызвавшее текущее исключение.</span><span class="sxs-lookup"><span data-stu-id="d12a6-127">The exception that is the cause of the current exception.</span></span>

#### <span data-ttu-id="d12a6-128">EdgeNotFoundException</span><span class="sxs-lookup"><span data-stu-id="d12a6-128">EdgeNotFoundException</span></span> 

<span data-ttu-id="d12a6-129">Инициализирует новый экземпляр класса EdgeNotFoundException с указанным сообщением об ошибке.</span><span class="sxs-lookup"><span data-stu-id="d12a6-129">Initializes a new instance of the EdgeNotFoundException class with a specified error message.</span></span>

> <span data-ttu-id="d12a6-130">общедоступный [EdgeNotFoundException](#edgenotfoundexception)(строковое сообщение)</span><span class="sxs-lookup"><span data-stu-id="d12a6-130">public [EdgeNotFoundException](#edgenotfoundexception)(string message)</span></span>

##### <span data-ttu-id="d12a6-131">Параметры</span><span class="sxs-lookup"><span data-stu-id="d12a6-131">Parameters</span></span>
* `message` <span data-ttu-id="d12a6-132">Сообщение об ошибке, объясняющее причину исключения.</span><span class="sxs-lookup"><span data-stu-id="d12a6-132">The error message that explains the reason for the exception.</span></span>

#### <span data-ttu-id="d12a6-133">EdgeNotFoundException</span><span class="sxs-lookup"><span data-stu-id="d12a6-133">EdgeNotFoundException</span></span> 

<span data-ttu-id="d12a6-134">Инициализирует новый экземпляр класса EdgeNotFoundException с указанным сообщением об ошибке и ссылкой на внутреннее исключение, которое является причиной этого исключения.</span><span class="sxs-lookup"><span data-stu-id="d12a6-134">Initializes a new instance of the EdgeNotFoundException class with a specified error message and a reference to the inner exception that is the cause of this exception.</span></span>

> <span data-ttu-id="d12a6-135">общедоступный [EdgeNotFoundException](#edgenotfoundexception)(строковое сообщение, исключение Inner)</span><span class="sxs-lookup"><span data-stu-id="d12a6-135">public [EdgeNotFoundException](#edgenotfoundexception)(string message, Exception inner)</span></span>

##### <span data-ttu-id="d12a6-136">Параметры</span><span class="sxs-lookup"><span data-stu-id="d12a6-136">Parameters</span></span>
* `message` <span data-ttu-id="d12a6-137">Сообщение об ошибке, объясняющее причину исключения.</span><span class="sxs-lookup"><span data-stu-id="d12a6-137">The error message that explains the reason for the exception.</span></span> 

* `inner` <span data-ttu-id="d12a6-138">Исключение, вызвавшее текущее исключение.</span><span class="sxs-lookup"><span data-stu-id="d12a6-138">The exception that is the cause of the current exception.</span></span>

